### ISSUE TITLE
I did not successfully compile your code

### ISSUE THREAD

**Issue by wangk98** [AlirezaShamsoshoara/Fire-Detection-UAV-Aerial-Image-Classification-Segmentation-UnmannedAerialVehicle#4]
Hello, When I run the code of "mode = 'training“, the following error appears.
![QQ图片20210306171917](https://user-images.githubusercontent.com/74230385/110202292-8bc9ec00-7ea2-11eb-857e-6a0ab60b6903.jpg)

Do you know how to solve it?

**Comment by AlirezaShamsoshoara** [comment]
Hello @wangk98, thanks for raising this issue. This case for the plot only works if you train your model with the METRIC parameters:
https://github.com/AlirezaShamsoshoara/Fire-Detection-UAV-Aerial-Image-Classification-Segmentation-UnmannedAerialVehicle/blob/473e289f0f9de127161bedd837567df923e68cd4/training.py#L42-L52

However, in the current version, only, accuracy was used as a metric. Hence, if you make these changes, there won't be any issue:
In plotdata.py, change line#21
https://github.com/AlirezaShamsoshoara/Fire-Detection-UAV-Aerial-Image-Classification-Segmentation-UnmannedAerialVehicle/blob/473e289f0f9de127161bedd837567df923e68cd4/plotdata.py#L21

to this:

`(fig, ax) = plt.subplots(1, 1, figsize=(13, 13))`

and comment out lines# 34 - 45 (inclusive).

I will make these changes in a new branch and I will submit them. Let me know if there is still any issues.

Good luck!

**Comment by wangk98** [comment]
Sorry to disturb you again. After I changed it according to the above method, the following error was reported:
![IMG_20210307_210558](https://user-images.githubusercontent.com/74230385/110240903-fce3cf00-7f88-11eb-9bf3-c9f0e124d106.JPG)
Then I change line#21(in plotdata.py) to  (fig, ax) = plt.subplots(2, 1, figsize=(13, 13),squezee=False)
the following error was reported:
![IMG_20210307_210558](https://user-images.githubusercontent.com/74230385/110241013-94e1b880-7f89-11eb-9699-a7b0ce3d9af4.JPG)
and I do not know how to solve it

**Comment by hyojun-rtdata** [comment]
Hello, thank you for you Great research.
I compile your code I have same error.

I changed code and commet lines.

![image](https://user-images.githubusercontent.com/80024811/119350276-1bc94600-bcda-11eb-9d8d-1cffc87aecf7.png)

**Comment by cryptedwcoder** [comment]
@hyojun-rtdata @wangk98 @AlirezaShamsoshoara Did you solve it ?

**Comment by MArslan95** [comment]
@AlirezaShamsoshoara 
<img width="952" alt="Screenshot 2024-05-14 164712" src="https://github.com/AlirezaShamsoshoara/Fire-Detection-UAV-Aerial-Image-Classification-Segmentation-UnmannedAerialVehicle/assets/59657976/ab3d80f5-6c5f-4927-8612-f2dec5a8a0b0">
Hi I hope you doing well I ran your code In my system while running. I get  the val_loss as nan I tried to find the sol but didn't find anything regarding this so please if you know the reason  let me know thank you

**Comment by SHUBH090298** [comment]
@MArslan95 did u solve it ? i have the same error. also, the validation accuracy is stagnant.

**Comment by joy-cxy** [comment]
> did u solve it ? i have the same error. also, the validation accuracy is stagnant.

 did u solve it ? i have the same error. also, the validation accuracy is stagnant.
